specs/0104-documentation-overhaul.md

```diff
@@ -0,0 +1,54 @@
+- Feature Name: Documentation Overhaul
+- Created: 2020-02-20
+- Associated Bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1616869
+
+# Summary
+
+Rebuild Pontoon's documentation to make it more accessible, more exhaustive and easier to maintain.
+
+# Motivation
+
+Pontoon's documentation is incomplete, sometimes out-dated, and spread out. Currently, we can find it on:
+
+- Technical documentation - https://mozilla-pontoon.readthedocs.io/
+- User documentation - https://mozilla-l10n.github.io/localizer-documentation/tools/pontoon/
+- L10n drivers documentation - https://mozilla-l10n.github.io/documentation/tools/pontoon/
+
+This might work to some extent for folks in the Mozilla community (thanks to l10n drivers knowing about this and being able to point users at the right places), but it is not easy to handle for external users, or people new to Pontoon.
+
+It is also unclear who is responsible for maintaining these documentations. That can lead to big delays between a change in a feature and its associated documentation being updated.
+
+# Feature explanation
+
+## Target audiences
+
+- **Localizer** — User of Pontoon wanting to learn how to efficiently translate and review projects. Looks for detailed feature documentation.
+- **Administrator** — User of Pontoon wanting to learn how to manage projects and teams. Looks for detailed administration tasks documentation.
+- **Project Owner** — Anyone wanting to make their project translated into several languages. Looks for internationalization and deployment documentation.
```

Comment:
I'm not sure we should include this target audience. If I understand this correctly, I'd assume this to be very specific to each org deploying Pontoon.

Reply:
Also, is this deploying Pontoon or projects localized with Pontoon?

Reply:
This covers deploying Pontoon as well as prerequisites on the side of the project to be localized (i.e. information about supported l10n frameworks, file formats and VCS systems, required access, file and folder structure):
* https://mozilla-pontoon.readthedocs.io/en/latest/user/localizing-your-projects.html (without Adding a new project section, which is *Administrator*)
* https://mozilla-pontoon.readthedocs.io/en/latest/admin/deployment.html
* https://mozilla-pontoon.readthedocs.io/en/latest/admin/maintenance.html